### DISCUSSION TITLE
Location Services for iOS mobile app

### DISCUSSION THREAD

**Discussion by ManciniXO** [phantom/docs#112] [General]
Working on an app centered around geolocation and Phantom mobile app for iOS currently doesn’t have Location Services so our app doesn’t work with Phantom on iOS. Would be great to get implemented before Breakpoint this year!

**Comment by adamdelphantom** [comment]
@ManciniXO thanks for posting this from our discussion on twitter. I will keep you updated as we make progress to allowing location services through our mobile browsers.
